pkg/ctl/utils/utils.go

```diff
@@ -7,7 +7,8 @@ import (
 )
 
 var (
-	clusterConfigFile = ""
+	clusterConfigFile string
+	output            string
```

Comment:
Please move the `output` variable to `export.go` where it is used.

Reply:
again, that follows existing code for consistency:
https://github.com/weaveworks/eksctl/blob/master/pkg/ctl/get/get.go#L11